### DISCUSSION TITLE
Relicense to AGPL3?

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#5554] [RFC]
Hi all,

@drewvolz, @drewthor, @StoDevX/all-about-olaf, @rye, @mattk410, @hannesmcman, @erichkauffman, @elijahverdoorn 

Does anyone have any objections to relicensing this project from MIT to AGPLv3?

The main difference is that under the AGPL, any institution who wants to use the app has to make their source code available upon request. 

I, personally, think that's a fine requirement. I like open source software. 

If I could get each account that I mentioned above to reply to this issue with a 👍 or 👎, and / or a text comment, that'd be great!

- Hawken

**Comment by drewvolz** [comment]
What do you think (👍 or 👎) @hannesmcman and @elijahverdoorn? Revisiting this now that it's been a year.

**Reply by drewvolz** [reply]
Hawken reached out to Hannes and Elijah on 8/29/22. We received positive responses which are included below:

Elijah:

> I believe that it would be a fine fit for AAO. If you and those that are actively working on the project believe that the change is worth making, go forward with my blessing on the change.

Hannes:

> That sounds good to me!

**Comment by hawkrives** [comment]
This has been completed!
